ngi_reports/common/ign_sample_report.py

```diff
@@ -62,11 +60,6 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.parse_picard_metrics()
         self.parse_deliveries()
 
-        self.create_aggregate_statistics()
-
-        self.LOG.debug('Plotting graphs')
-        self.make_plots()
```

Comment:
Note that I moved this function call to the node-specific classes. Unless other code uses the common class directly and expects this method to be called from here, this should be fine...